src/Symfony/Component/Mailer/Command/MailerDebugCommand.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Mailer\Exception\TransportExceptionInterface;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+
+/**
+ * Helps making sure your mailer provider is operational.
+ *
+ * @author Guillaume MOREL <me@gmorel.io>
+ *
+ * @final
+ */
+class MailerDebugCommand extends Command
+{
+    /** {@inheritdoc} */
+    protected static $defaultName = 'debug:mailer:email:send';
```

Comment:
I think `debug:mailer:send-email` might make more sense. I don't think we'll have multiple debug command in the `debug:mailer:email` namespace

Reply:
Good catch :+1:

Reply:
I think this shouldn't be in the debug namespace, this is actually sending an email using the whole infrastructure, so why not just naming it `mailer:send-email`? Specially if it will also handle files.

Reply:
Indeed, if we handle file input it will be more than just a debug mailer command

Reply:
I removed the `debug` concept as no relevant anymore